integration/fuse/src/test/java/alluxio/fuse/AlluxioFuseFileSystemTest.java

```diff
@@ -175,6 +175,21 @@ public void create() throws Exception {
         .build());
   }
 
+  @Test
+  public void createWithLengthLimit() throws Exception {
+    String c256 = "0123456789ABCDEF" + "0123456789ABCDEF"
```

Comment:
Manually creating this `c256` is error prone. How about using `String.join("", Collections.nCopies(16, "0123456789ABCDEF"));` like https://stackoverflow.com/a/6857936/3161750?
Also about creating a `c255` and assert success?

Reply:
good idae. Thanks.